dialog/information_test.go

```diff
@@ -16,3 +18,42 @@ func TestDialog_MinSize(t *testing.T) {
 
 	assert.Less(t, label.Width, dialogContent.Width)
 }
+
+func TestDialog_InformationCallback(t *testing.T) {
+	d := NewInformation("Information", "Hello World", test.NewWindow(nil))
+	tapped := make(chan bool)
+	d.SetOnClosed(func() { tapped <- true })
+	d.Show()
+
+	information := d.(*dialog)
+	assert.False(t, information.win.Hidden)
+	go test.Tap(information.dismiss)
+	func() {
+		select {
+		case <-tapped:
+		case <-time.After(1 * time.Second):
+			assert.Fail(t, "Timed out waiting for button tap")
+		}
+	}()
+	assert.True(t, information.win.Hidden)
+}
+
+func TestDialog_ErrorCallback(t *testing.T) {
+	err := errors.New("Error message")
+	d := NewError(err, test.NewWindow(nil))
+	tapped := make(chan bool)
+	d.SetOnClosed(func() { tapped <- true })
+	d.Show()
+
+	information := d.(*dialog)
+	assert.False(t, information.win.Hidden)
+	go test.Tap(information.dismiss)
+	func() {
+		select {
+		case <-tapped:
+		case <-time.After(1 * time.Second):
```

Comment:
That's way too long IMO.
```suggestion
		case <-time.After(100 * time.Millisecond):
```

Reply:
Oh, when using `test.Tap` you don't have to do concurrency, it's synchronous. However, using a `chan` and the `select` makes the test independent from the implementation of `test.Tap`.

Reply:
It looks like this is still 1 second

Reply:
Sorry, fixed.